modules/items/server/policies/items.server.policy.js

```diff
@@ -30,6 +30,10 @@ exports.invokeRolesPolicies = function () {
       resources: '/api/categories',
       permissions: '*'
     },
+    {
+        resources: '/api/pdf',
+        permissions: '*'
+    },
```

Comment:
These changes don't make that much sense. We could just render an html link to the offsite pdf storage, so we don't need any routing or policies at all. Keep the schema change but this will be dead code unless we're doing this a different way than I thought we were.

Reply:
This was a change I had to make so that the PDF links showed up in the list view.